Stripe/PublicHeaders/STPPaymentContext.h

```diff
@@ -104,6 +104,15 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic, copy)NSString *paymentCurrency;
 
 /**
+ *  If you support Apple Pay, you can optionally set the PKPaymentSummaryItems you want to display here.
+ *  If nil, a single summary item will be automatically generated using paymentAmount and your companyName.
+ *  @see PKPaymentRequest for more information
+ 
+ *  @warning If you set this array, the last summary item's amount must match the value you set for paymentAmount or an assert will be thrown. 
```

Comment:
Nice. It seems likely that a user may not realize the last item needs to be the total amount, so the assert is a nice guard. I was briefly considering the alternative where we use the last line item as the `paymentAmount`, but that's probably more surprising behavior.